src/recon_buildblock/BackProjectorByBin.cxx

```diff
@@ -173,7 +173,7 @@ back_project(DiscretisedDensity<3,float>& density,
      iter != viewgrams.end();
      ++iter)
       {
-    ViewSegmentNumbers vs(iter->get_view_num(), iter->get_segment_num());
+    ViewSegmentTOFNumbers vs(iter->get_view_num(), iter->get_segment_num());
```

Comment:
`get_timing_pos_num()`

Reply:
This is apart of `#ifdef STIR_PROJECTORS_AS_V3`. No TOF support so leaving.

Reply:
ok, but can you create an issue on UCL/STIR to deprecate any functions in this ifdef and call `error` here?